public/scripts/Table.js

```diff
@@ -38,6 +38,7 @@ var Table = React.createClass({
         return data.map(function(item) {
             var cells = cols.map(function(colData) {
                 return <td> {item[colData.key]} </td>;
+              {item[colData.key]}.sort(sort_by('name1', asec));
```

Comment:
- Think about what action will trigger the sort function.....
 - you need to use a [onClick trigger](https://facebook.github.io/react/docs/interactivity-and-dynamic-uis.html)  on the headerCell to do this.
- Also look into state management in react. [Identifying State scope](https://facebook.github.io/react/docs/thinking-in-react.html#step-4-identify-where-your-state-should-live)